app/controllers/tickets_controller.rb

```diff
@@ -20,11 +20,11 @@ def notify_owner
 
   def send_ticket_notification(owner=false)
     message = TicketDispenser::Message.find(notification_params[:message_id])
-    sender_email = message.details[:sender_email]
+    sender_email = message.ticket.messages.first.details[:sender_email]
```

Comment:
That error you were getting related to the `to` column has to do with this snippet. `:sender_email` is only stored on the first message. When we send a reply, it's not certain that we will have access to the sender's email. This walks back up to the first message and grabs the sender email from there.

Alternatively, what about sending just the first message from the front end? I think it might make the front-end code a bit more complex but it'll make less database calls.

Reply:
Hmm... this seems fine. This particular DB call is going to be trivial and fast.